services/core-api/app/date_time_helper.py

```diff
@@ -0,0 +1,20 @@
+from dateutil.relativedelta import relativedelta
+
+
+def get_duration_text(start_date, end_date):
```

Comment:
Did you move this logic to the BE or duplicate the logic on the BE? Is there a chance they will ever return different values?

Reply:
Mekhti wrote tests to ensure that the function works correctly. It's the same logic as the FE.